draft-ietf-trans-rfc6962-bis.md

```diff
@@ -156,6 +156,11 @@ used to send various CT log artifacts.
 Logs are network services that implement the protocol operations for submissions
 and queries that are defined in this document.
 
+\[RFC Editor: please update 'RFCXXXX' to refer to this document,
+once its RFC number is known, through the document. Note that the use
+in the ASN.1 Appendix "RFCRFCXXXX" is deliberate. \]
```

Comment:
I think `Note that the use in the ASN.1 Appendix "RFCRFCXXXX" is deliberate` was applicable to your first draft, but is no longer applicable.  If you agree, please remove this sentence.